app/store/state/entities/selectors.js

```diff
@@ -2,18 +2,26 @@ import { get } from "lodash";
 
 import { NotFoundEntity, isValid } from "../../../dataDefinitions/defects";
 
-export const getEntity = (state, entityType, id) =>
-  get(state, `entities.${entityType}.${id}`) === undefined
-    ? NotFoundEntity
-    : state.entities[entityType][id];
+export const bindState = mapStateToProps => (state, ...args) =>
+  mapStateToProps.bind(state)(...args);
 
-export const getEntityValue = (state, entityType, id, key) =>
```

Comment:
there is no should'nt be a get Entity value but a get entity and a get value
getValueFrom is the get value